app/helpers/federation_helper.rb

```diff
@@ -94,4 +98,22 @@ def request_portals_names
       content_tag(:span, federated_portal_name(name), style: color ? "color: #{color}" :  "", class: color ? "" : "text-primary")
     end.compact
   end
+
+  def federatation_enabled?
+    params[:federate] || params[:portals]
+  end
+
+  def portal_button(name: nil , color: nil , light_color: nil, link: nil, tooltip: nil)
```

Comment:
this needs to be used in the browse page, we can't have multiple UI component to show the same thing.